app/controllers/application_controller.rb

```diff
@@ -11,6 +12,24 @@ def cors_preflight
 
   private
 
+  rescue_from "Exception" do |exception|
+    log_error(exception)
+    case exception
+    when VA::API::Common::Exceptions::BaseError
+      render json: { errors: exception.errors }
+    else
+      render json: { errors: VA::API::Common::Exceptions::InternalServerError.new(exception).errors }
+      # FIXME: do we need to re-raise the exception as long as we are logging it and rendering something?
```

Comment:
We could have a cloudwatch metric here to increment the exception so it can be part of an alert or dashboard.